docs/docs/dev/documentation/zoneegress.md

```diff
@@ -0,0 +1,59 @@
+# Zone Egress
+
+In the situation, when Kuma is deployed in a [multi-zone](../deployments/multi-zone.md),
+and you want to achieve isolation of outgoing traffic (to services in other 
+zones or [external services](../policies/external-services.md) in the local zone),
+you can use `ZoneEgress` proxy.
+
+This proxy is not attached to any particular workload. Instead, it's bound to
+that particular zone.
+
+When Zone Egress is present in the zone:
+* All requests that are sent from local data plane proxies to the ones in other
+  zones will be directed through the local Zone Egress instance, which then will
+  direct the traffic to the proper instance of the Zone Ingress.
+* All requests that are sent from local data plane proxies to [external services](../policies/external-services.md)
+  available within the Zone will be directed through the local Zone Egress
+  instance.
+
+The `ZoneEgress` entity includes a few sections:
+
+* `type`: must be `ZoneEgress`.
```

Comment:
Isn't that just the type of the entity in the end?

Reply:
I'm not sure what do you mean to be honest